src/main/kotlin/no/nav/familie/klage/behandling/domain/Behandling.kt

```diff
@@ -22,14 +22,27 @@ data class Behandling(
     val sporbar: Sporbar = Sporbar(),
     val resultat: BehandlingResultat = BehandlingResultat.IKKE_SATT,
     val vedtakDato: LocalDateTime? = null,
-    val eksternFagsystemBehandlingId: String,
+    @Embedded(onEmpty = Embedded.OnEmpty.USE_EMPTY)
+    val påklagetVedtak: PåklagetVedtak,
     val klageMottatt: LocalDate,
     val behandlendeEnhet: String,
     val eksternBehandlingId: UUID = UUID.randomUUID(),
     @Column("henlagt_arsak")
     val henlagtÅrsak: HenlagtÅrsak? = null
 )
 
+data class PåklagetVedtak(
+    val eksternFagsystemBehandlingId: String?,
+    @Column("paklaget_vedtak")
+    val påklagetVedtakstype: PåklagetVedtakstype
+)
+
+enum class PåklagetVedtakstype {
+    Vedtak,
+    UtenVedtak,
+    IkkeValgt
```

Comment:
Burde det være camelcase? 
```suggestion
    VEDTAK,
    UTEN_VEDTAK,
    IKKE_VALGT
```

Reply:
Burde det være noe som validerer at man valgt noe annet enn `ikke_valgt` før man kan lagre formkrav eller noe slik?

Reply:
Må nok ha noe der, men det bør vente til vi faktisk har frontend+backend-integrasjonen på plass.